src/Action.hs

```diff
@@ -104,12 +110,6 @@ instance PersistM r => ActionPersist r (Action r) where
 instance MonadIO (Action r) where
     liftIO = Action . liftIO
 
-instance PersistM r => PersistM (Action r) where
```

Comment:
I prefer the first better. And write different and explicit authorization when we use the `persistent` combinator.